### DISCUSSION TITLE
posts/why-elon-musk-cant-buy-bluesky/

### DISCUSSION THREAD

**Discussion by giscus** [seanthegeek/seanthegeek.github.io#3] [giscus]
# posts/why-elon-musk-cant-buy-bluesky/

Developers, data nerds, and VCs have made something special that even Elon Musk can’t buy

https://seanthegeek.net/posts/why-elon-musk-cant-buy-bluesky/

**Comment by gavincompton** [comment]
I think you mean corporation, not cooperation. (Great post, though!)

**Reply by seanthegeek** [reply]
D'oh! Fixed. Thanks

**Comment by w-dib** [comment]
Found your post where you talked about centering the bluesky embed. Glad it worked!

Just a little typo: Amjad Masad is the founder of the Replit, not *Resplit*

**Comment by Feyter** [comment]
I am very much disagreeing with the headline of the article.

A scenario in that Bluesky is sold to someone like Elon musk is very much possible despite the fact that it's open source.

The point is, that Bluesky decided to not be a truly decentralized federated service like PeerTube, Lemmy, Mastodon and many other.

Bluesky with it's Relay structure is designed in a way that needs a centralized instance. Perhaps it's possible in theory to setup multiple relays but in reality everything is designed in a way that only the one centralized relay operated by Bluesky (the company) will exist. If you buy the company operating this centralized point you own Bluesky (the network). 

So from this argument alone people would be better fitted in the "Fediverse" which already is a federation of different and  completely independent services. ActivityPup is designed in a way that enables true decentralization.

Perhaps people just want the 1to1 Twitter experience, perhaps Bluesky for the moment is driven by good people, but history repeating itself is very much possible with Bluesky.

I also recommend the following article:
https://dustycloud.org/blog/how-decentralized-is-bluesky/

**Reply by seanthegeek** [reply]
I just now saw this reply, over a month later. The point of the article was to highlight the fact that Bluesky is run by a Public Benefit Corporation. If Musk or anyone else with the intent of turning it into another X with closed algorithms that stifled speech of people they don't like, that would be illegal under the law in which Bluesky was founded.
